### ISSUE TITLE
Feature request: Add onboarding message for new members of #accessibility Slack

### ISSUE THREAD

**Issue by lukefretwell** [CivicActions/accessibility#410]
**For whom is this feature?**
A clear and concise user story.
As a CivicActioner joining the #accessibility Slack channel, I would like a welcome message with key instructions/information, so that I can best understand how to get engaged.

**Describe the solution you'd like**

- [x]  Set up onboarding for new channel members: https://slack.com/slack-tips/automatically-onboard-new-channel-members

Ideas:

* Join the meetings /join + /calendar
* Take W3C training

**Comment by lukefretwell** [comment]
![Screen Shot 2021-05-24 at 11 38 11 AM (2)](https://user-images.githubusercontent.com/1569463/119392702-efcfa580-bc84-11eb-9bdf-ca4c5310f3ab.png)
